### PR TITLE
Display feature pets on org home page [558]

### PR DESCRIPTION
**Pull request by VonTeacher**
# 🔗 Issue
[Display feature pets on org home page #558](https://github.com/rubyforgood/pet-rescue/issues/558)

# ✍️ Description
- [x] Created instance variable in the `Organizations::HomeController#index` action to sample from the adoptable_pets (though I'm not sure the pattern matches what is used in other controllers).
- [x] Replaced four static pet cards on the `/<org>/home` page with a sample of the adoptable pets from that organization.

# 📷 Screenshots/Demos
## Before:
image
## After:
Screen Shot 2024-04-11 at 17 59 11

### CODE REVIEW

**Comment by kasugaijin** [comment]
FYI tests failures and need to run the linter.

File: test/controllers/organizations/home_controller_test.rb [hunk 1]
```diff
@@ -2,15 +2,23 @@
...
 class Organizations::HomeControllerTest < ActionDispatch::IntegrationTest
   setup do
-    @organization = create(:organization)
+    @pets = create_list(:pet, 4, :with_image)
+    set_organization(@pets.first.organization)
   end
 
   context "GET #index" do
     should "render index template" do
-      get home_index_path(script_name: "/#{@organization.slug}")
```
Comment (kasugaijin) [inline test/controllers/organizations/home_controller_test.rb:10]: I think removing this script name will cause issues in the tests.

**Comment by kasugaijin** [comment]
@all-contributors
please add @VonTeacher for code.

**Comment by kasugaijin** [comment]
@VonTeacher let me know if you have any questions!

**Comment by kasugaijin** [comment]
@VonTeacher hey are you still able to finish this? Please let me know in the next couple of days. I usually re-assign issues that sit without communication for a week or more. Thanks :)

**Comment by VonTeacher** [comment]
Thanks for the reminder and apologies for the delay. I'll spend some time this weekend, or will drop off if not able.

**Comment by kasugaijin** [comment]
@VonTeacher No worries at all! Just making sure you have what you need and still able to work on it.

File: test/controllers/organizations/home_controller_test.rb [hunk 2]
```diff
@@ -12,7 +13,14 @@ class Organizations::HomeControllerTest < ActionDispatch::IntegrationTest
       get home_index_path(script_name: "/#{@organization.slug}")
 
       assert_response :success
-      assert_select "title", text: "#{@organization.name} | \"Pet Rescue\""
+      assert_select "title", text: "#{ActsAsTenant.current_tenant.name} | \"Pet Rescue\""
```
Comment (kasugaijin) [inline test/controllers/organizations/home_controller_test.rb:16]: This is failing. Although we assert specific dom elements in this test suite, I am recommending we steer away from it as it couples the test suite too tightly with the dom and leads to test failures whenever we touch the dom in the future. 

I personally don't think we need to test this particular feature, but if we wanted to test something, I would just test the index action on the controller returns a collection of 4 pets that belong to the organization set in the test context. Or, just remove the test changes altogether.

File: test/controllers/organizations/home_controller_test.rb [hunk 3]
```diff
@@ -4,6 +4,7 @@ class Organizations::HomeControllerTest < ActionDispatch::IntegrationTest
   setup do
     @organization = create(:organization, :with_page_text)
     @organization.save
+    @pets = create_list(:pet, 4, :with_image, organization_id: @organization.id)
```
Comment (kasugaijin) [inline test/controllers/organizations/home_controller_test.rb:7]: Should we need this? The Index action should be performing a query to retrieve 4 pets from the given Org, and we should be able to access that via the instance var within a controller test?

**Comment by kasugaijin** [comment]
@VonTeacher let me know if you can update re my feedback above. If not, I can also finish this off if you're busy. Let me know :)

**Comment by kasugaijin** [comment]
@maebeale yes we can it just needs the tests removing or updating. I personally don’t think m this needs to be tested.

**Comment by meg-gutshall** [comment]
Resolved by #784

**Pull request closed without merging**